### ISSUE TITLE
New feature suggestion: Qibla Compass

### ISSUE THREAD

**Issue by akmal-kamarudin** [farhan-helmy/ratemysurau#46]
Can we make a new feature to show qibla direction just in case if there is no available surau near to the user.

Implementation example:
https://qiblafinder.withgoogle.com/intl/en/onboarding

![photo_6174701884359687643_y](https://github.com/farhan-helmy/ratemysurau/assets/74535874/1b869902-82ee-4833-b7fa-a275d3682443)
![photo_6174701884359687642_y](https://github.com/farhan-helmy/ratemysurau/assets/74535874/c8f21c5c-d797-40ca-a412-dee08db7bf20)

For starters we can just make a link to redirect to the URL link given. Shall I take on this task?

**Comment by ismi-abbas** [comment]
Good idea too. There's a table contain the compass coordinate i've added before, I'm not sure if you could redirect it to the compass at. To point to accurate direction.

**Comment by akmal-kamarudin** [comment]
> Good idea too. There's a table contain the compass coordinate i've added before, I'm not sure if you could redirect it to the compass at. To point to accurate direction.

I haven't figured out the logic yet but thanks for the heads up, I'll try to work on this later on.
